### ISSUE TITLE
backup to git

### ISSUE THREAD

**Issue by beasteers** [ysde/grafana-backup-tool#251]
Being able to use github as a storage backend would be nice because then you'd be able to view diffs, do branching/PRs, etc.

**Comment by nodens** [comment]
Agreed. I'm looking into grafana-backup right now, an while I plan to use the --no-archive flag and use a wrapper of some sort to commit, tag and push, there is still a need for an archive to restore.

It would be nice to at least have the option to restore from a folder (of course, git integration allowing to restore a specific commit/tag/branch/point in time would be awesome, but that'd be a start).
